docs/configuration.md

```diff
@@ -40,4 +40,18 @@ class MyWallet extends WalletBase {
    echo $user->wallet->helloWorld();
 ```
 This same method above, can be used to extend the base `Transfer` and `Transaction` models and registering the extended models in the configuration file.
+### Changing wallet decimal places
+
+You can change the default wallet decimal places, from your extended wallet model by defining a `decimal_places` attribute on the model. This is useful when working with fractional numbers
+
```

Comment:
Hello. I highly do not recommend this method. Better this way:

config/wallet.php
```php
    /**
     * Base model 'wallet'.
     */
    'wallet' => [
        ....
        'creating' => [
            'decimal_places' => 18,
        ],
       ....
    ],
```

Reply:
Okay. It will be nice to document what the creating array actually does and the possible values.

Reply:
Maybe you can edit the pull request

Reply:
Yes, maybe, but I can’t give you any advice on the timing. If you do this earlier, it will be great.

Reply:
effected the necessary changes